app/helpers/dynamodb.py

```diff
@@ -13,6 +13,7 @@
 from app.settings import AWS_DB_REGION_NAME
 from app.settings import AWS_DB_TABLE_NAME
 from app.settings import AWS_S3_BUCKET_NAME
+from app.settings import DEFAULT_CLIENT_VERSION
```

Comment:
Probably add a short explanation of this env var in the README.md#deployment-configuration

Reply:
P.S.: I guess the default value for `DEFAULT_CLIENT_VERSION` will be set in the k8s configs, right?

Reply:
@hansmannj no the default client version is fixed and hardcoded to `0.0.0`. It is used for previous drawing that are generated by the legacy viewer. This is not a configuration I just used a global variable because it is set in several place in code.